recipes/level-zero/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, apply_conandata_patches, copy, export_conandata_patches
+
+
+class LevelZeroConan(ConanFile):
+    name = "level-zero"
+    license = "MIT"
+    homepage = "https://github.com/oneapi-src/level-zero"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "OneAPI Level Zero Specification Headers and Loader"
+    topics = ("api-headers", "loader", "level-zero", "oneapi")
+    package_id_non_embed_mode = "patch_mode"
```

Comment:
```suggestion
```

I would ask to remove this custom package id mode, it will result in missing binaries in CCI because minor is the default mode used.